src/lib/components/Media/media.js

```diff
@@ -0,0 +1,11 @@
+import { createMedia } from '@artsy/fresnel';
+
+export const AppMedia = createMedia({
```

Comment:
Nice! Was this suggested by Semantinc-UI?

Reply:
Yes, there is a "migration" guide -> Semantic-Org/Semantic-UI-React#4008